examples/animation/animated_fox.rs

```diff
@@ -316,7 +316,7 @@ fn spawn_particle<M: Material>(
 
 #[derive(Component)]
 struct Particle {
-    lifeteime_timer: Timer,
+    lifetime_timer: Timer,
```

Comment:
I believe this change is safe to do, and looks contained to this file. If there should be any debate around it I can revert it.

Reply:
this is fine